backend/core/src/listings/listings.service.ts

```diff
@@ -107,6 +115,90 @@ export class ListingsService {
     }
   }
 
+  public async listIncludeExternal(
+    bloomJurisdictions: string[],
+    params: ListingsQueryParams
+  ): Promise<ListingIncludeExternalResponse> {
+    return {
+      local: await this.list(params),
+      external: await this.listExternal(bloomJurisdictions, params),
+    }
+  }
+
+  private async listExternal(
+    bloomJurisdictions: string[],
+    params: ListingsQueryParams
+  ): Promise<JurisdictionIdToExternalResponse> {
+    if (bloomJurisdictions == null || bloomJurisdictions.length == 0) {
+      return {}
+    }
+    // Transform params to Bloom-friendly ones
+    const bloomParamsNoJurisdiction = this.copyParamsForBloom(params)
+    // begin build all http requests
+    const httpRequests: Promise<AxiosResponse>[] = []
+    for (const jurisdiction of bloomJurisdictions) {
+      const newFilter = [
+        ...(bloomParamsNoJurisdiction.filter || []),
+        {
+          jurisdiction: jurisdiction,
+          $comparison: Compare["="],
+        },
+      ]
+      httpRequests.push(
+        firstValueFrom(
+          this.httpService
+            .get(
+              this.configService.get<string>("BLOOM_API_BASE") +
+                this.configService.get<string>("BLOOM_LISTINGS_QUERY"),
+              {
+                params: {
+                  ...bloomParamsNoJurisdiction,
+                  filter: newFilter,
+                },
+                paramsSerializer: (params) => {
+                  return qs.stringify(params)
+                },
+              }
+            )
+            .pipe(
+              catchError((error) => {
+                if (error.response) {
+                  throw new NotFoundException("Bloom /listings did not return a result")
+                } else {
+                  // If there is no response, there was most likely a problem on our end.
+                  throw new InternalServerErrorException()
```

Comment:
Do we want to handle external API failures so that the client request returns some data, instead of a 500? Is it practical to map an error message per external call?

Reply:
I recommend logging an error in the application; we'll want to keep track of these types of failures as a metric in the medium to long term. 

I think it makes sense to return at least the internal results, even if the externally hosted sites are failing.

Reply:
New behavior (note that this returns a 200 mostly bc I don't see a way to change the http status without throwing an exception, but the client can figure it out via missing bloomJurisdictions): 
![image](https://user-images.githubusercontent.com/10789523/230141295-9fa840aa-d206-4415-a03a-009063455849.png)